patches/rrweb@2.0.0-alpha.17.patch

```diff
@@ -0,0 +1,482 @@
+diff --git a/dist/rrweb.js b/dist/rrweb.js
+index 12f4ad3c7f897fe8a9ee2f4595dcf37f302d17c5..65b80aa52851aa8d57bfa218e639a850f1b19cb7 100644
+--- a/dist/rrweb.js
++++ b/dist/rrweb.js
+@@ -15007,7 +15007,7 @@ class MediaManager {
+     }
+   }
+   addMediaElements(node2, timeOffset, mirror2) {
+-    if (!["AUDIO", "VIDEO"].includes(node2.nodeName)) return;
++    if (!this.isSupportedMediaElement(node2)) return;
+     const target = node2;
+     const serializedNode = mirror2.getMeta(target);
+     if (!serializedNode || !("attributes" in serializedNode)) return;
+@@ -15060,6 +15060,7 @@ class MediaManager {
+     timeOffset,
+     mutation
+   }) {
++    if (!this.isSupportedMediaElement(target)) return;
```

Comment:
Suggested fix in https://github.com/rrweb-io/rrweb/pull/1462 that allows us to remove the previous `target?.pause()` patches